app/models/address.rb

```diff
@@ -135,8 +135,23 @@ def to_a
   end
 
   def fetch_county_fips_code
-    us_county = BenefitMarkets::Locations::CountyFips.where(state_postal_code: state, county_name: county).first
-    us_county ? us_county.county_fips_code : nil
+    state_abbrev = state.blank? ? '' : state.upcase
+    county_name = county.blank? ? '' : county.titlecase
```

Comment:
how does this interact with counties that have 2 words in the name?  Do we have examples of that?  How do we store such county names in our DB?  Does what this is doing match that?

Either way I'd like to see a two-word spec.

Reply:
irb(main):012:0> ::BenefitMarkets::Locations::CountyZip.all.map(&:county_name).uniq
=> ["Androscoggin", "Aroostook", "Cumberland", "Franklin", "Hancock", "Kennebec", "Knox", "Lincoln", "Oxford", "Penobscot", "Piscataquis", "Sagadahoc", "Somerset", "Waldo", "Washington", "York"]

These are county we have in ME

Reply:
@TreyE we don't have 2 words county in ME, and the reason we are checking for `titlecase` here because for ME entire data set is imported, and we saved without formatting in DB.

And FIPS code we use only for ME, other exchanges don't use this.

If someone enters an address through UI we don't have to run any formats on the county name, as it saved correctly in the backend.